dash/development/build_process.py

```diff
@@ -138,20 +141,20 @@ def bundles(self, build=None):
 
             shutil.copyfile(
                 self._concat(self.npm_modules, scope, name, subfolder, filename),
-                self._concat(self.build_folder, target),
+                self._concat(self.deps_folder, target),
             )
 
         _script = "build:dev" if build == "local" else "build:js"
         logger.info("run `npm run %s`", _script)
-        os.chdir(self._concat(self.build_folder, os.pardir, "dash-renderer"))
+        os.chdir(self.main)
         run_command_with_process("npm run {}".format(_script))
```

Comment:
not exactly sure, but shouldn't we wait for this command to execute?

Reply:
`run_command_with_process` does [wait](https://github.com/plotly/dash/blob/ca4fcf5d0a209cd2865bd7ddba52952c83f2126e/dash/_utils.py#L221)